pkgs/sops-install-secrets/main.go

```diff
@@ -952,8 +977,11 @@ func installSecrets(args []string) error {
 	if err := handleModifications(isDry, manifest.Logging, manifest.SymlinkPath, *secretDir, manifest.Secrets); err != nil {
 		return fmt.Errorf("Cannot request units to restart: %w", err)
 	}
-	// No need to perform the actual symlinking
+	// No need to perform the actual symlinking, just remount as readonly
 	if isDry {
+		if err := ensureSecretFs(manifest.SecretsMountPoint, true, keysGid); err != nil {
```

Comment:
I still have not figured out why this sometimes fail on some hosts. Can we keep it behind a feature flag until the issue is resolved?